js/web/components/HomePage.js

```diff
@@ -181,7 +184,7 @@ const HomePage = () => {
           sx={{ paddingBottom: { md: '140px', xs: '30px' } }}
         >
           Artists receive 100% of their sales. The only fee is a one-time
-          payment (on average ~$4/release) that covers the storage and
+          payment (${`${(NinaProgramActionCost[NinaProgramAction.ReleaseInitViaHub] * solPrice).toFixed(2)}`} + $0.005/MB) that covers the storage and
```

Comment:
showing `NaN` for cost
![Screen Shot 2022-09-05 at 1 58 37 PM](https://user-images.githubusercontent.com/22109520/188498075-002c0f8c-a9a2-4373-bbb6-96bc62c3e46a.png)

Reply:
Ah good catch - changed casing of the enum to WHATEVER_THIS_CASE_IS_CALLED

will fix